Application/DependencyInjection.cs

```diff
@@ -13,6 +13,8 @@ public static void AddApplication(this IServiceCollection services)
         services.AddTransient<CompensationCreationCommand>();
         services.AddTransient<CompensationsService>();
         services.AddTransient<ICompensationsQuery, CompensationsQuery>();
+        services.AddTransient<IPersonalCompensationsQuery, PersonalCompensationsQuery>();
+        services.AddTransient<ITenantCompensationsQuery, TenantCompensationsQuery>();
```

Comment:
You shouldn't rename it after Tenant, it is by default. Then, doesn't make sense to call it that way.